server/scsynth/SC_PortAudio.cpp

```diff
@@ -341,9 +338,25 @@ bool SC_PortAudioDriver::DriverSetup(int* outNumSamples, double* outSampleRate)
 	mDeviceInOut[0] = paNoDevice;
 	mDeviceInOut[1] = paNoDevice;
 	if (mWorld->hw->mInDeviceName)
-		GetPaDeviceFromName(mWorld->hw->mInDeviceName, mDeviceInOut);
+		GetPaDeviceFromName(mWorld->hw->mInDeviceName, &mDeviceInOut[0], true);
+	if (mWorld->hw->mOutDeviceName)
+		GetPaDeviceFromName(mWorld->hw->mOutDeviceName, &mDeviceInOut[1], false);
+
+	#ifdef _WIN32
+	//if one device is specified, let's try to open another one on matching api
```

Comment:
not necessary to change it right now but in the future please add a space after `//` in comments